src/migrations/20190705144236-create-user-table.ts

```diff
@@ -0,0 +1,32 @@
+import { DataTypes, QueryInterface } from "sequelize";
+
+export default {
+  up: (queryInterface: QueryInterface): Promise<void> => {
+    return queryInterface.createTable("User", {
+      createdAt: {
+        allowNull: false,
+        type: new DataTypes.DATE()
+      },
+      deletedAt: new DataTypes.DATE(),
+      fc: {
+        allowNull: false,
+        field: "FC",
+        primaryKey: true,
+        type: new DataTypes.STRING()
+      },
+      name: {
```

Comment:
```suggestion
      firstName: {
```

Reply:
do we need to store this information in the db? can't we keep it in the session?

Reply:
Keeping this information in session is not enough because it could be needed by some asynchronous processes running independently of the user session, like the verification of a digital signature.